tools/openblas_support.py

```diff
@@ -323,6 +323,9 @@ def test_version(expected_version=None):
 
     data = threadpoolctl.threadpool_info()
     if len(data) != 1:
+        if platform.python_implementation() == 'PyPy':
+            print(f"Check broken in CI on PyPy, data is: {data}")
+            return
```

Comment:
Maybe not worth the effort to explore, but this is strange.

Reply:
It failed in CI, then I did go through the effort of setting up a PyPy dev env locally, but there it worked fine. So I'd like to punt on this at least for now.